app/controllers/subscribers_from_account_token_controller.rb

```diff
@@ -0,0 +1,79 @@
+class SubscribersFromAccountTokenController < ApplicationController
+  before_action :validate_params
+
+  class AccountsDown < RuntimeError; end
+  class TokenInvalid < RuntimeError; end
+
+  rescue_from AccountsDown do |exception|
+    render json: { error: "Could not fetch subscriber",
+                   details: exception.message },
+           status: :internal_server_error
+  end
+
+  rescue_from TokenInvalid do |exception|
+    render json: { error: "Could not fetch subscriber",
+                   details: exception.message },
+           status: :forbidden
+  end
+
+  def from_token
+    address = expected_params.require(:address)
+    token = expected_params.require(:token)
+
+    validate_token!(address, token)
+
+    subscriber = find_subscriber(address)
+
+    render json: { subscriber: subscriber }
+  end
+
+private
+
+  def authorise
+    authorise_user!("manage_subscriptions_from_account")
```

Comment:
Do we need a special permission for this? I found it a bit confusing because this code is both authorising the requesting app and the user. Since the call to accounts is to validate a token I don't think we'd care too much if this end point was called by a different authenticated user would we?